self-hosting/README.md

```diff
@@ -0,0 +1,304 @@
+# HOSTING THE CHT APP
+
+The community health toolkit core framework has been packaged into a docker container to make it portable and easy to install. It is available from [dockerhub](https://hub.docker.com/r/medicmobile/medic-os). To learn more how to work with docker you could follow the tutorial [here](https://docker-curriculum.com/#getting-started) and the cheat sheet [here](https://www.docker.com/sites/default/files/d8/2019-09/docker-cheat-sheet.pdf).  
```

Comment:
"community health toolkit" should be capitalized: "Community Health Toolkit" (or shortened to "CHT" if you want ; )